website/blog/2024-05-24-Agent/index.mdx

```diff
@@ -0,0 +1,142 @@
+
+On the other hand, we
+don't have to stop there. We can think about multi-agent system as a way to multiply the
+power of a single agent. We can connect them with other agents to accomplish bigger goals.
+
+### Benefits of multi-agents
+
+There are at least two types of applications that benefit from multi-agent systems.
+
+- Single-agent interface.
+Developers often find that they need to extend the system with different capabilities, tools, etc. And if
+they implement that single agent interface with a multi-agent architecture, they can often increase the capability to
+handle more complex tasks or improve the quality of the response. One example is complex data analytics.
+It often requires agents of different roles to solve a task. Some agents are good at retrieving the data and
+presenting to others. Some other agents are good at running deep analytics and providing insights. We can also have agents which can do critics and suggest more actions. Or agents that can do planning, and so on.
+Usually, to accomplish a complex task, one can build these agents with different roles.
+> If you don't know about Chi Wang and Microsoft Research's work, please check it out. I want to give a real world production use case for Skypoint AI platform client Tabor AI https://tabor.ai AI Copilot for Medicare brokers - selecting a health plan every year for seniors (65 million seniors have to do this every year in the US) is a cumbersome and frustrating task. This process took hours to complete by human research, now with AI agents 5 to 10 minutes without compromising on quality or accuracy of the results. It's fun to see agents doing retail shopping etc. where accuracy is not that mission critical. AI in regulated industries like healthcare, public sector, financial services is a different beast, this is Skypoint AI platform (AIP) focus.
+>
+> > Tisson Mathew, CEO @ Skypoint
+
+- Multi-agent interface. For example a chess game needs to have at
+least two different players. A
+football game involves even more entities. Multi-agent debate
+and social
+simulation are good examples too.
+
+![leadership](img/leadership.png)
+
+### Cost of multi-agents
+
+Will using multiple agents increase the cost, latency, and chance of failures, compared to using a single agent? It depends on how the multi-agent system is designed, and surprisingly, the answer can be the opposite.
+
+- Even if the performance of a single agent is good enough, you may also want to make this single agent teach some other relatively cheaper agent so that they can become
+better with low cost. [EcoAssistant](/blog/2023/11/09/EcoAssistant) is a good example of combining GPT-4 and GPT-3.5 agents to reduce the cost while improving the performance even compared to using a single GPT-4 agent.
+- A recent use case reports that sometimes using multi-agents with a cheap model can outperform a single agent with an expensive model:
```

Comment:
I think it is important to acknowledge that multi-agent systems can be expensive for really advanced use cases. 
But a comment that might be able to be helpful is to compare the cost of MAS vs human cost something like

"sure very complex Mult-Agent systems with leading frontier models are expensive, but compared to having humans accomplish the same task they are exponentially more affordable"

I can give a specific quote about this regarding our savings / efficiency bump for our use case at BetterFutureLabs if it would be helpful for you